.github/workflows/continuous-integration-workflow.yml

```diff
@@ -32,8 +32,8 @@ jobs:
         with:
           repository: kokkos/ci-containers/${{ matrix.config.dockerfile }}
           registry: ghcr.io
-          username: kokkos
-          password: ${{ secrets.DOCKER_TOKEN }}
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
```

Comment:
ghcr.io only works with personal access tokens, see https://docs.github.com/en/free-pro-team@latest/packages/guides/migrating-to-github-container-registry-for-docker-images#authenticating-with-the-container-registry

Reply:
OK. Just removing username and password seems to work.